sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -71,7 +71,7 @@ describe("Random scheme in the endpoint from connection string", function(): voi
   async function beforeEachTest(testClientType: TestClientType) {
     sbClient = createServiceBusClientForTests();
     entities = await sbClient.test.createTestEntities(testClientType);
-    await sbClient.close();
+    await sbClient.test.after();
```

Comment:
We need this before and after? Shouldn't it just be in the afterEach?

Reply:
Hmmm, in the very next line there is another SBC (sbClient2) which is supposed to be the primary one for the two tests in this test suite, and I wanted to clear this other sbClient right away.

On second thoughts, if we ever consider the plan of clearing test entities in the `test.after()` call, this gets affected. So, I'm moving this to afterEach.